### DISCUSSION TITLE
Open style sheet editor in new window without tab bars

### DISCUSSION THREAD

**Discussion by paul1149** [openstyles/stylus#1315] [Ideas]
I would like to see the style sheet editor open in a new window (which it already does) without tab bars. My native Vivaldi set up has two vertical tab bars that take up a lot of room. I can hide them with Ctrl-F11, but it would be nice if it would be done automatically. Thanks.

**Comment by tophf** [comment]
Stylus already has an option for this but Vivaldi doesn't implement such functionality - it seems to be unable to create such windows internally - so you can ask them on their forum to fix it.

![image](https://user-images.githubusercontent.com/1310400/129513845-a39b9c5c-0be0-48cd-8fca-f75e65e97549.png)

**Comment by paul1149** [comment]
Ok, I see it does work in Slimjet. I will ask at Vivaldi. Thank you.
